src/controllers/itemController.js

```diff
@@ -0,0 +1,159 @@
+const Item = require('../models/Item');
+const User = require('../models/User');
+const { StatusCodes } = require('http-status-codes');
+
+// Add an item to the database. This will be updated to require userId in the next PR.
+const createItem = async (req, res) => {
+    try {
+        const item = await Item.create(req.body);
+        res.status(StatusCodes.CREATED).json({msg:'The item has been created.', item});
+
+    } catch (error) {
+        console.log(error.message);
```

Comment:
@mariasantiago01 Question: is it Express/Node practice to leave console.log statements in? On the front-end, we tend to strip them out before committing code.

Reply:
That's a great question! I was leaving them in just in case it -was- standard practice to communicate errors in the console, in addition to the json (in this case). I just looked into it and I guess it is not, so I will have them removed in the next PR.

Reply:
@mariasantiago01 thanks for investigating that. Let's remove all the console.log statements in this PR. All you have to do is make the changes then push them up. You won't have to open a new PR to make that change. I'll approve + merge then because everything else looks great

Reply:
Thank you, sounds good. I just pushed the changes.